risk-map/yaml/controls.yaml

```
@@ -80,6 +80,23 @@ controls:
     risks:
       - DP
       - UTD
+    mappings:
+      mitre-atlas:
+        - AML.M0013
```

This seems to be code signing https://atlas.mitre.org/mitigations/AML.M0013 which isn't really a control for training data sanitization...

I'd suggest we want AML.M0007 https://atlas.mitre.org/mitigations/AML.M0007 which is Sanitize Training Data

```suggestion
        - AML.M0007
```